tests/integration/test_github.py

```diff
@@ -6,6 +6,17 @@
 from ogr.abstract import PRStatus
 from ogr.services.github import GithubService
 
+def requirements_met():
```

Comment:
if you add this in conftest.py, pytest will inject it into every test module, no need to have it three times

Reply:
@TomasTomecek that sounds interesting, I've did not know that it is possible.

But there are and could be various requirements, eg. test_github needs to have set ``GITHUB_TOKEN`` and ``GITHUB_USER``, and pagure tests has ``PAGURE_USER`` and token, and eg. for upstream pagure you theoretically need to have 3 tokens, although for dowstream you need just one. so that is reason why this is probably better solution as I think.

Reply:
`conftest.py` is IMHO used for fixtures only, pytest does not import functions from `conftest.py` into other modules - but I might be wrong.
But we can put `skip_tests()` into `conftest.py` and import it explicitly to other modules
```python
from .conftest import skip_tests
```

And in each module we would just specify the requirements
```python
pytestmark = skip_tests(["GITHUB_TOKEN", "GITHUB_USER"])
```